src/deluge/io/midi/midi_engine.cpp

```diff
@@ -218,13 +218,9 @@ MidiEngine::MidiEngine() {
 	lastStatusByteSent = 0;
 	currentlyReceivingSysExSerial = false;
 	midiThru = false;
-	midiFollow = false;
 	for (auto& midiChannelType : midiEngine.midiFollowChannelType) {
 		midiChannelType.clear();
 	}
-	midiFollowChannelType[util::to_underlying(MIDIFollowChannelType::SYNTH)].channelOrZone = 15;
```

Comment:
these should be initialized to none (255)

Reply:
for (auto& midiChannelType : midiEngine.midiFollowChannelType) {
		midiChannelType.clear();
	}
	
	this code takes care of that :)